### PR TITLE
submissions: add acquisition source for literature

### PR DESCRIPTION
**Pull request by harunurhan**
* INSPIR-2266

### CODE REVIEW

File: inspirehep/submissions/views.py [hunk 1]
```diff
@@ -23,20 +23,48 @@
...
+            internal_uid=int(current_user.get_id()),
+        )
+
+        orcid = self._get_user_orcid()
+        if orcid:
+            acquisition_source["orcid"] = orcid
+
+        return acquisition_source
+
+    def _get_user_orcid(self):
```
Comment (drjova) [inline inspirehep/submissions/views.py]: we don't need ``_`` here since we moved it

Comment (harunurhan) [reply inspirehep/submissions/views.py]: it's still bit a of private thing though.

Comment (drjova) [reply inspirehep/submissions/views.py]: it's similar to ``get_acquisition_source``

File: tests/integration/submissions/test_submissions_views.py [hunk 2]
```diff
@@ -262,6 +261,8 @@ def test_new_literature_submit(app, api_client, create_user_and_token, requests_
         == history.headers["Authorization"]
     )
     assert history.url == f"{app.config['INSPIRE_NEXT_URL']}/workflows/literature"
+    assert "datetime" in post_data["data"]["acquisition_source"]
+    del post_data["data"]["acquisition_source"]["datetime"]
```
Comment (drjova) [inline tests/integration/submissions/test_submissions_views.py]: why we delete that?

Comment (drjova) [reply tests/integration/submissions/test_submissions_views.py]: https://github.com/inspirehep/inspirehep/blob/d3968f24b810a91ef1711daf4600976a7af6dcf7/tests/integration/records/serializers/test_latex.py#L11

Comment (harunurhan) [reply tests/integration/submissions/test_submissions_views.py]: alternative would be to to mock `time` (not sure if we have it) and test MOCKED time. I don't think this has a lot of value compared to effort needed. So checking if it's there or not it's fair IMO.

**Pull request merged**